### DISCUSSION TITLE
Argo Rollouts Controller HPA

### DISCUSSION THREAD

**Discussion by vbrinza** [argoproj/argo-rollouts#3114] [Q&A] [answered]
We have situations when we are doing changes in tens of Argo Rollouts simultaneously which causes the Controller to OOM.
I see the Argo Rollouts Controller chart does not have an HPA template.
Wanted to hear the community/maintainers opinions on introducing one.

**Comment by kostis-codefresh** [comment, marked as answer]
This is probably an issue for https://github.com/argoproj/argo-helm
